jdaviz/cli.py

```diff
@@ -76,11 +75,37 @@ def main(filepaths=None, layout='default', instrument=None, browser='default',
         args += ['--auto-restart']
     else:
         args += ['--production']
-    cli(['run', 'jdaviz.solara',
-         '--theme-loader', 'plain',
-         '--theme-variant', theme,
-         '--host', host,
-         '--port', port] + args)
+    run_solara(host=host, port=port, theme=theme, browser=browser)
```

Comment:
is it possible to maintain support for `hotreload` (which was packed into `args`) or should we remove that from the CLI entirely?

Reply:
Good catch. Actually, the `--auto-restart` is only needed when developing with solara (see https://solara.dev/documentation/advanced/reference/reloading)

If you do not pass `--production`, it will still auto reload (vue files, CSS, markdown and python files).

In any case, I'll fix it!